python/data/ArtusConfigs/Includes/settingsSampleStitchingWeights.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import logging
+import Artus.Utility.logger as logger
+log = logging.getLogger(__name__)
+
+import re
+import json
+import copy
+import Artus.Utility.jsonTools as jsonTools
+import Kappa.Skimming.datasetsHelperTwopz as datasetsHelperTwopz
+import importlib
+
+def build_config(nickname):
+  config = jsonTools.JsonDict()
+  datasetsHelper = datasetsHelperTwopz.datasetsHelperTwopz("Kappa/Skimming/data/datasets.json")
```

Comment:
what is the datasetsHelper needed for in this function?

Reply:
I intended it to be a basic part of a module but one could actually comment it out and it would still be easily reactivated if needed. In the recent commit it is now uncommented in every file where it is not needed